Add Rattles + fixes

JamesVanBoxtel opened the pull request:
Added all rattles assets including meta data
Fixed a couple files that didn't have 2x tag
Deleted unused garbage reference files
added documentation on telegraph images as I needed it to double check I was at double resolution

JamesVanBoxtel committed:
Add Rattles + fixes

Added all rattles assets including meta data
Fixed a couple files that didn't have 2x tag
Deleted unused garbage reference files
added documentation on telegraph images as I needed it to double check I was at double resolution

Endaris reviewed (commented):
Haven't looked through the metadata yet but we'll need to update COPYING-ASSETS for the new assets.
Since I wasn't involved in the process it will be difficult for me to verify the source of each contribution, so I'd ask you to use that opportunity to double check that they are correct.

Endaris committed:
add info to COPYING-ASSETS
add comment about dc:Contributor
fix a phrase

Endaris commented:
I added the items to COPYRIGHT-ASSETS as provided with a small change for brevity.

Rattles telegraph has no copyright notice in its metadata and there's notably some weird history tag with a bunch of useless information.
The metadata for Rattles's garbage files should include DonavinDraws in the Contributor field.
Rattles's SFX somehow have their genre tagged as Blues which...is not an issue but a bit confusing.

Endaris committed:
update THANKS.md

JamesVanBoxtel committed:
Updates

* Added Donavin and me info to Rattles garbage with a command like this
exiftool -XMP-dc:Creator="JamBox" -XMP-cc:AttributionName="JamBox" -XMP-cc:License="https://creativecommons.org/licenses/by-sa/4.0/"  -XMP-dc:Rights="Copyright, JamBox, 2024. This work is licensed under the Creative Commons Attribution ShareAlike 4.0 International License. To view a copy of this license, visit https://creativecommons.org/licenses/by-sa/4.0/" -XMP-dc:Contributor="Donavin Draws" -overwrite_original bot*@2x.png doubleface@2x.png face@2x.png filler1@2x.png filler2@2x.png flash@2x.png left@2x.png pop@2x.png right@2x.png top*@2x.png
* Added me as info for Esme and Magellan garbage
* Added Gaster as Creater, Copyright and License to portraits and icons of all three characters with a command like this
exiftool -XMP-dc:Creator="Gaster" -XMP-cc:AttributionName="Gaster" -XMP-cc:License="https://creativecommons.org/licenses/by-sa/4.0/"  -XMP-dc:Rights="Copyright, Gaster, 2025. This work is licensed under the Creative Commons Attribution ShareAlike 4.0 International License. To view a copy of this license, visit https://creativecommons.org/licenses/by-sa/4.0/" -overwrite_original icon@2x.png portrait2@2x.png portrait@2x.png
* Fixed Rattles sfx to have "Video Game" as genre
* Added author info to telegraph of all three characters

Endaris committed:
remove xmpMM, ICC and GIMP metadata from Rattles's combo telegraph files

Endaris committed:
add license metadata to garbage_land and garbage_match sfx

Endaris committed:
fix THANKS.md

Endaris reviewed (approved):
Did some more fixes, looks good now I think
